src/azure-cli/azure/cli/command_modules/search/commands.py

```diff
@@ -46,10 +46,14 @@ def load_command_table(self, _):
         g.command('list', 'list_by_resource_group')
         g.show_command('show', 'get')
         g.command('delete', 'delete', confirmation=True)
```

Comment:
You should remove these two lines if you want to use the `delete` and `show` commands from aaz directly

Reply:
done